site/content/en/docs/Getting started/installation.md

```diff
@@ -4,94 +4,102 @@ linkTitle: "Install Kustomize"
 date: 2022-02-27
 weight: 10
 description: >
-  Installing Kustomize
+  Kustomize can be installed in a variety of ways
 ---
 
-Kustomize can be installed in a variety of ways.
+## Go Source
 
-## Binaries
-Binaries are available for Linux, MacOS and Windows, across a variety of architectures.
+Requires [Go] to be installed.
 
-You can see the full list of releases here on the [Github releases page](https://github.com/kubernetes-sigs/kustomize/releases).
+### Install the kustomize CLI from source without cloning the repo
 
-### Quick install
-Get the latest build of Kustomize for your platform.
 ```bash
-curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+GOBIN=$(pwd)/ GO111MODULE=on go get sigs.k8s.io/kustomize/kustomize/v4
 ```
-You can also pass optional `version` and `target_dir` arguments to the script:
+
+### Install the kustomize CLI from local source with cloning the repo
+
 ```bash
-curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" 4.4.1 $HOME/bin  | bash
+# Need go 1.13 or higher
+unset GOPATH
+# see https://golang.org/doc/go1.13#modules
+unset GO111MODULES
+
+# clone the repo
+git clone git@github.com:kubernetes-sigs/kustomize.git
+# get into the repo root
+cd kustomize
+
+# Optionally checkout a particular tag if you don't
+# want to build at head
+git checkout kustomize/v4.5.2
+
+# build the binary
+(cd kustomize; go install .)
+
+# run it
+~/go/bin/kustomize version
```

Comment:
```suggestion
# this assumes your Go bin (generally GOBIN or GOPATH/bin) is on your PATH
kustomize version
```

(`~/go` is the default value of GOPATH)

Alternatively, we could link to https://go.dev/doc/code and/or https://pkg.go.dev/cmd/go#hdr-Environment_variables instead of speculating where the binary might have gone. WDYT?